src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.After.targets

```diff
@@ -19,6 +19,7 @@ This file is imported *after* the Microsoft.NET.Sdk/Sdk.targets.
   <Import Project="..\tools\Xamarin.Android.Bindings.ClassParse.targets" />
   <Import Project="..\tools\Xamarin.Android.Bindings.Maven.targets" />
   <Import Project="..\tools\Xamarin.Android.Bindings.JavaDependencyVerification.targets" />
+  <Import Project="..\tools\Microsoft.Android.Sdk.Bindings.Gradle.targets" />
```

Comment:
Rather than having each target be conditional based on `@(AndroidGradleProject)` not being empty, would it instead make sense to move that condition to this import?

```xml
 <Import Project="..\tools\Microsoft.Android.Sdk.Bindings.Gradle.targets" Condition=" '@(AndroidGradleProject->Count())' != '0' " />
```

Reply:
maybe, it all depends on when things get evaluated by MSBuild. I would test it and see if it works 😄

Reply:
Looks like this type of condition won't work on an import:
> error MSB4099: A reference to an item list at position 3 is not allowed in this condition

Reply:
Yeah, the `<Import/>` & `<PropertyGroup>` pass happens before the `<ItemGroup>` pass...

I think a `Condition` on each target is probably simplest.